File: Practice/Autum/C++/21. Индекс массы тела/21. Индекс массы тела/21. Индекс массы тела.cpp
#include <iostream>
using namespace std;

double bmi( double height, double weight) {
    double k;
    height = height / 100;
    k= weight / (height * height);
    return k;
}

void print_bmi(double k) {
    double bmi(double height, double weight);
    if (k < 18.5)
        cout << "Underweight";
    else if (k < 25 )
        cout << "Normal";
    else if (k  < 30)
        cout << "Overweight";
    else 
        cout << "Obesity";
}
int main()
{
    double weight, height;
    cout << "Write you height and weight:";
    cin >> height>>weight;
    print_bmi(bmi(height, weight));

}

File: Practice/Autum/Python/21. Индекс массы тела.py
def bmi(weight: float, height: float) -> float:
    height = height / 100
    return weight / (height ** 2)

def print_bmi(bmi: float) -> float:
   if (bmi < 18.5):
       print("Underweight")
   elif (bmi < 25):
       print("Normal")
   elif (bmi < 30):
       print("Overweight")
   else:
       print("Obesity")
  
height, weight = map(float, input('Write you height and weight: ').split(' ')); 
print_bmi(bmi(weight, height))


